### ISSUE TITLE
indentation problem leading to missing TssEnrichment.txt

### ISSUE THREAD

**Issue by michalstolarczyk** [databio/pepatac#104]
I'm getting an error when running the [test example](https://github.com/databio/pepatac/tree/master/examples/test_project):
```
File "/home/mjs5kd/code//pepatac/tools/pyTssEnrichment.py", line 93
    continue
           ^
TabError: inconsistent use of tabs and spaces in indentation
```

[PEPATAC_log.txt](https://github.com/databio/pepatac/files/3142917/PEPATAC_log.txt)

**Comment by michalstolarczyk** [comment]
did _not_ fail when run with python 2.7.15

**Comment by nsheff** [comment]
this was fixed in 4f2288e
